tracer/test/benchmarks/Benchmarks.Trace/AgentWriterBenchmark.cs

```diff
@@ -12,6 +13,7 @@
 
 namespace Benchmarks.Trace
 {
+    [JsonExporterAttribute.FullCompressed]
```

Comment:
If you do it that way, we may forget to apply the attribute on new benchmarks. You should be able to add it to all benchmarks by default like we do the datadog exporter: https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/benchmarks/Benchmarks.Trace/Program.cs#L25